packages/destination-actions/src/destinations/salesforce/__tests__/case.test.ts

```diff
@@ -0,0 +1,273 @@
+import nock from 'nock'
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import Destination from '../index'
+import { API_VERSION } from '../sf-operations'
+
+const testDestination = createTestIntegration(Destination)
+
+const settings = {
+  instanceUrl: 'https://test.com'
+}
+const auth = {
+  refreshToken: 'xyz123',
+  accessToken: 'abc123'
+}
+
+describe('Salesforce', () => {
+  describe('Case', () => {
+    it('should create a case record', async () => {
+      nock(`${settings.instanceUrl}/services/data/${API_VERSION}/sobjects`).post('/Case').reply(201, {})
+
+      const event = createTestEvent({
+        event: 'Identify',
+        traits: {
+          description: 'This is test description'
+        }
+      })
```

Comment:
I see that these test payloads were based on prior art, so our bad here! I would recommend some changes to ensure that the events conform to our [spec](https://segment.com/docs/connections/spec/):

```suggestion
      const event = createTestEvent({
        type: 'track',
        event: 'Create new case',
        properties: {
          description: 'This is test description'
        }
      })
```